### PR TITLE
Basiq-34:Remove 'Select Account' screen

### PR DESCRIPTION
**Pull request by albatahmet**
For Basiq-34 task, from account verification steps, steps 4 and 5 were disabled, after entering username and password for account, redirected to home page.

disabled pages;
AccountVerificationFormStep4SelectAccount.js
AccountVerificationFormStep5Summary.js

### CODE REVIEW

File: components/AccountVerificationForm/AccountVerificationFormProvider.js [hunk 1]
```diff
@@ -306,10 +306,6 @@ function useBasiqConnection({ currentStep, userId }) {
       return;
     }
     if (completed) {
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormProvider.js]: You can remove this if condition block completely, there is no need to leave it as empty @albatahmet

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js [hunk 2]
```diff
@@ -10,18 +11,37 @@ export function AccountVerificationFormStep3LoadingSteps() {
   // State for managing hiding/showing of the resume in background modal
   const [isResumeModalOpen, openResumeModal, closeResumeModal] = useTernaryState(false);
 
-  const { basiqConnection, goForward } = useAccountVerificationForm();
+  const { basiqConnection ,finish,updateAccountVerificationFormState} = useAccountVerificationForm();
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: Please use spaces correctly @albatahmet

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js [hunk 3]
```diff
@@ -10,18 +11,37 @@ export function AccountVerificationFormStep3LoadingSteps() {
...
     setJobId(newJobId);
-  }, [setJobId])
+  }, [])
 
+  const { data } = useAccountsData({
+    userId: sessionStorage.getItem("userId"),
+  });
+
+  const errorOrNoData = error || !data || data.length === 0;
+  async function autoSelectedAccount () {
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: Why did you create a function to select an account automatically ? You can put this logic in fetchAccounts method and set currentAccount

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js [hunk 4]
```diff
@@ -58,8 +70,47 @@ export function AccountVerificationFormStep3LoadingSteps() {
...
+  const [data, setData] = useState();
+  const [error, setError] = useState();
+  
+  const { updateAccountVerificationFormState } = useAccountVerificationForm();
+
+  const fetchAccounts = useCallback(() => {
+    axios
+      .get('/api/accounts', { params: { userId } })
+      .then(res => {
+        console.log(res.data);
```
Comment (MahmutKasimoglu) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: seems like a forgotten console.log

Comment (albatahmet) [reply components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: I forgot it, thank you very much

**Review by MahmutKasimoglu — APPROVED** [review]

**Review by k-cem-fd — APPROVED** [review]

**Pull request merged**